server/api/api_test.go

```diff
@@ -168,3 +170,120 @@ func TestSetLink(t *testing.T) {
 		})
 	}
 }
+
+func TestGetLink(t *testing.T) {
+	for _, tc := range []struct {
+		name         string
+		prevLinks    []autolink.Autolink
+		autoLinkName string
+		expectStatus int
+		expectReturn string
+	}{
+		{
+			name:         "get the autolink",
+			autoLinkName: "test",
+			prevLinks: []autolink.Autolink{{
+				Name:     "test",
+				Pattern:  ".*1",
+				Template: "test",
+			}},
```

Comment:
these values are the same in both test cases. I think we can use them directly and remove them from here

Reply:
check everywhere